cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,18 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden. in_entrance_exam is inherited metadata, all children will have it.
```

Comment:
@cahrens let me explain what i am doing here: 
1) hiding the header contents of subsection('sequential') for an entrance exam (not for verticals/units). 
2) is_entrance_exam is set True for chapter (not for subsections and nor for verticals). 
3) in_entrance_exam is True for all children even for parent too (you are right) But here we are interested to know that xblock is subsection ('sequential') of entrance exam ('chapter') but NOT for verticals/units so that is why i am checking that the parent of xblock ('sequential') is entrance exam.